src/core/threadasm.S

```diff
@@ -406,21 +406,32 @@ fiber_switchContext:
 
 .text
 .align  2
-.global fiber_switchContext
+#ifdef __APPLE__
+.global _fiber_switchContext
+_fiber_switchContext:
+#else
```

Comment:
I guess this name decoration business should look like this:
```asm
.global CSYM(fiber_switchContext)
.type   CSYM(fiber_switchContext), %function
CSYM(fiber_switchContext):
```
It's used like this for the other functions below.

Reply:
Fixed.